tests/org.eclipse.swt.tests.win32/ManualTests/org/eclipse/swt/widgets/PatternWin32ManualTest.java

```diff
@@ -0,0 +1,51 @@
+package org.eclipse.swt.widgets;
+
+import org.eclipse.swt.graphics.Color;
+import org.eclipse.swt.graphics.Pattern;
+import org.eclipse.swt.internal.DPIUtil;
+
+public class PatternWin32ManualTest {
```

Comment:
Add a brief description and explain:
* What does this snippet test?
* Why was it added (instead of an automatic test)? Briefly explain why is it difficult to test automatically.
* What should the user see and expect when he/she runs this test? What should he/she validate/check?

Since this is a manual **test**, all these things should be documented so one can still test the code 6 months from now.

Reply:
Done